calebwagner/uteach-ilearn-server
# creating models

#### Changes Made
1. Added file ...
> `__init__.py`,
> `app_user.py`,
> `category.py`
> `comment.py`
> `connection.py`
> `meeting.py`
> `message.py`
> `post.py`
> `savepost.py`
... to `uteachilearnapi/models` directory.

1. Modified _each above model_ to include needed `fields` and imported all models to `__init__.py`.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-creating-models
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
- [ ] check for above newly created models
- [ ] ensure each model is imported into the `__init__.py` file

4. View code file.
   > Confirm file modifications are present as indicated above.
